### PR TITLE
Feature/embedder

### PR DESCRIPTION
**Pull request by Dremars**
New feature embedder tool for creating embed html and links.

Need to check:
- ~~translations~~ done
- ~~area view handling~~ Moved to another card. Will be added later
- ~~possibly embed preview optimization relation to option changes.~~ Seems fine now

### CODE REVIEW

**Comment by Dremars** [comment]
[Upotustyökalu-kokonaisuus - Mockup](https://trello.com/c/jaPJjJZy/536-upotusty%C3%B6kalu-kokonaisuus-mockup)

File: src/views/EmbedderView/components/EmbedController.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    <Paper className={classes.root} elevation={1}>
+      <InputBase
+        className={classes.input}
+        disabled={disabled}
+        value={value}
+        onChange={inputOnChange}
+      />
+      {
+        preText && (
+          <pre className={classes.iconButton} aria-label="Search">
```
Comment (mikkoisa) [inline src/views/EmbedderView/components/EmbedController.js:59]: Is aria-label supposed to be "search" here?

Comment (Dremars) [reply src/views/EmbedderView/components/EmbedController.js:59]: I'm not sure but seems like it's a placeholder since it's not using translations

Comment (Dremars) [reply src/views/EmbedderView/components/EmbedController.js]: Resolved in a093cdd

**Review by mikkoisa — APPROVED** [review]

**Pull request merged**